src/plugins/fcrypt/README.md

```diff
@@ -23,8 +23,9 @@ After the getstorage plugin has read the backend file, the plugin decrypts the b
 
 ## Security Considerations
 
-During decryption the plugin temporarily writes the decrypted plain text to the same directory as the original (encrypted) file.
-This is a vulnerability as an attacker might have access to the plain text for a short period of time (the time between pregetstorage and postgetstorage calls).
+During decryption the plugin temporarily writes the decrypted plain text to a configurable temporary directory (`/tmp` is used as default).
+This might be a vulnerability as an attacker might have access to the plain text for a short period of time (the time between pregetstorage and postgetstorage calls).
```

Comment:
Its not clear for me from this text that there are actually two problems:

1. A file is shortly visible in a filesystem. Protection: use a directly which is only readable by the user. Is this already addressed in this PR?
2. Bytes of private data might end up on the hard disc. Protection: tmpfs